lib/qs.rb

```diff
@@ -77,6 +81,10 @@ def self.dispatcher_job_name
     self.config.dispatcher_job_name
   end
 
+  def self.published_events
+    self.dispatcher_queue.enqueued_jobs.map(&:event)
```

Comment:
@jcredding we discussed having queue objects implement a `published_events` method and just demetering that here.